cime_config/namelist_definition_blom.xml

```diff
@@ -3863,15 +3863,16 @@
    <desc>File name (incl. full path) for atmopheric N-deposition data</desc>
   </entry>
 
-  <entry id="use_nuopc_ndep">
+  <entry id="use_coupler_ndep" modify_via_xml="HAMOCC_ATMNDEPC">
     <type>logical</type>
     <category>config_bgc</category>
     <group>config_bgc</group>
     <values>
       <value>.false.</value>
       <value comp_interface="nuopc">.true.</value>
+      <value hamocc_atmndepc="yes">.true.</value>
```

Comment:
How will this work now - the `use_coupler_ndep` is `.false.` by default, but `.true.` if either `comp_interface="nuopc"` or `hamocc_atmndepc="yes"`?
Have you tested what happens if `comp_interface="nuopc"` and `hamocc_atmndepc="no"? I guess the entry will still be `.true.` in this case.

Reply:
Hi @TomasTorsvik , good question. In the setup that I have currently available, I went though all your suggested combinations (via `xmlchange` and `preview_namelist`) and it works as expected - so if `comp_interface='nuopc'` and `hamocc_atmndep='no'`, `use_coupler_ndep=.true.`, if both are set, it results also in `.true.` and if `comp_interface=mct` and `hamocc_atmndep='yes'`, `use_coupler_ndep` is alo `.true.` (the latter now also tested in a short coupled simulation).

Reply:
In any case, it's currently not really safe using nuopc-coupled nitrogen deposition fields, since there still seems to be a persistent issue: #411 (and similar for dust #404) - I haven't looked into those thus far (and probably won't the next weeks).